adagrams/game.py

```diff
@@ -1,11 +1,135 @@
+import random
+
 def draw_letters():
-    pass
+ 
+    NUM_TILES = 10
+    letters_grabbed = []
+    letters_remaining_pool = LIST_OF_LETTERS.copy()
+    
+    for i in range(NUM_TILES):
+        # update the last index of the remaining pool list that changes everytime
+        # random.randint stop is inclusive, so -1 which reflect the size of the pool
```

Comment:
My opinion about comments is if your code is readable/concise/you used descriptively named variables and functions to convey your intent then you do not need to leave comments in the code.

As a fellow developer, I know what your code is doing if it is written in a readable way. I feel that if you need comments to explain what is happening, then the code should be refactored to be more clear.

That's my opinion and other instructors might feel differently or you may end up working on a team that is ok with comments, but they can clutter up your project. It's important to keep large code bases easy to read and understand.